doc/script_commands.txt

```diff
@@ -9031,16 +9031,21 @@ This command is used in all the item scripts for taming items. Running this
 command will make the pet catching cursor appear on the client connected to the
 invoking character, usable on the monsters with the specified pet ID number. It
 will still work outside an item script.
+If the <pet id> is PET_CATCH_UNIVERSAL it will return the monster target ID
```

Comment:
I know this is not your fault, but could you add the alternative names for those commands while we are cleaning up the documentation?
bpet's real name is birthpet and pet's real name is catchpet.
Thanks a lot!

Reply:
when i did use `catchpet` instead of pet in the item_db i get error
```
[Error]:
script error on db/import/item_db.txt line 70
    parse_line: expect command, missing function name or calling undeclared function
*   70 : { 'c'atchpet PET_CATCH_UNIVERSAL_ITEM; }
```

Reply:
wont we need to add them in the script.cpp too ?
```
BUILDIN_DEF(catchpet,"i"),
BUILDIN_DEF(birthpet,""),
```

Reply:
Yeah those are missing.